# StartRecordingSVO.srv

# request fields
bool start_recording # Set to false to stop.
string video_filename # Absolute filepath.

---
# response fields
bool success
